File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.java
class Solution {
    public int[] solution(String[] wallpaper) {
        int[] answer = {};
        
        int lux = 50, luy = 50;
        int rdx = 0, rdy = 0;
        
        for (int i = 0; i < wallpaper.length; i++) {
            for (int j = 0; j < wallpaper[i].length(); j++) {
                if (wallpaper[i].charAt(j) == '#') {
                    lux = Math.min(lux, i);
                    luy = Math.min(luy, j);
                    rdx = Math.max(rdx, i);
                    rdy = Math.max(rdy, j);
                }
            }
        }
        
        return new int[]{lux, luy, rdx + 1, rdy + 1};
    }
}

File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.py
def solution(wallpaper):
    answer = []
    
    lux, luy = 50, 50
    rdx, rdy = 0, 0
    
    for i in range(len(wallpaper)):
        for j in range(len(wallpaper[i])):
            if wallpaper[i][j] == '#':
                lux = min(i, lux)
                luy = min(j, luy)
                rdx = max(i, rdx)
                rdy = max(j, rdy)
    
    return [lux, luy, rdx + 1, rdy + 1]